contracts/src/libraries/token/v2/erc677.cairo

```diff
@@ -0,0 +1,79 @@
+use starknet::ContractAddress;
```

Comment:
doing a diff  between `v2/erc667.cairo` and `v1/erc667.cairo` will make the changes more apparent